### ISSUE TITLE
[bug] Fleet manager fail to capture equipment slotlevel

### ISSUE THREAD

**Issue by HKGPsy** [KC3Kai/KC3Kai#1712]
This is particularly obvious when I export it to deckbuilder

I have a similar issue a week ago, but that one is a direct export from the game panel to deckbuilder and it only fail to capture carrier plane slotlevel. In fleet manage of strategy room, it does not capture any slotlevel of any equipment.

**Comment by Javran** [comment]
duplicate of #1665 
that issue was marked as `wontfix` but I think this problem should be addressed eventually.
we can begin by updating `library/modules/Database.js` so that equipment improvement level and aircraft proficiency can be recorded in sortie history from now on... not sure whether any of @KC3Kai/developers knows how to update it?

**Comment by sinsinpub** [comment]
@ethrundr no, we didn't record the api_data directly, and even not record every properties of Fleet / Ship / Gear to save space of IndexDB.

See the method in [`Fleet.js`](https://github.com/KC3Kai/KC3Kai/blob/master/src/library/objects/Fleet.js#L807), I can add `stars` and `ace` easily. And I have added new property to store LBAS info sent to sortie/defend.
But question is: is it enough for us to just add `stars` and `ace` recording?

**Comment by Javran** [comment]
@sinsinpub I think LBAS confgs + item stars & aircraft proficiency should be enough.

**Comment by Javran** [comment]
Fixed by #1719, #1725

**Comment by Javran** [comment]
old sorties won't have improvement level & aircraft proficiency because we didn't record them. but new sorties after #1719 will.
